compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -413,23 +363,21 @@ readVersion = do
   minor <- readMinorVersion
   pure (Version major minor)
 
--- >>> parseVersion "1.dev"
--- Just (Version {versionMajor = V1, versionMinor = PointDev})
--- >>> parseVersion "1.15"
--- Just (Version {versionMajor = V1, versionMinor = PointStable 15})
--- >>> parseVersion "1.garbage"
+-- >>> parseVersion "2.dev"
+-- Just (Version {versionMajor = V2, versionMinor = PointDev})
+-- >>> parseVersion "2.15"
+-- Just (Version {versionMajor = V2, versionMinor = PointStable 15})
+-- >>> parseVersion "2.garbage"
 -- Nothing
 parseVersion :: String -> Maybe Version
 parseVersion = headMay . map fst . readP_to_S readVersion
 
--- >>> show (VersionReq (\case V1 -> noMinorVersion; V2 -> allV2MinorVersions))
--- "VersionReq (\\case V1 -> Empty; V2 -> Inclusive_ PointDev PointDev)"
+-- >>> show (VersionReq (\V2 -> allMinorVersions))
+-- Variable not in scope: allV2MinorVersions :: Range MinorVersion
```

Comment:
I haven't checked this is the actual output but it's probably something like this. Just a comment so not a huge problem

```suggestion
-- "VersionReq (\\case V2 -> Inclusive_ (PointStable 1) PointDev)"
```

Reply:
Indeed! I forgot to click that one after I fixed the error message. Done.